src/Command/CreateCommand.php

```diff
@@ -90,7 +90,6 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
-        $this->migrator->setIO($io);
```

Comment:
Commands should run `setIO()`, otherwise `ConsoleMigrationInformer` will be not work.

Reply:
It is initialized inside `MigrationService`

https://github.com/yiisoft/db-migration/pull/221/files#diff-af4e2edfa363b5a64f1326262e8bb035653c7dd417f4f9ecc355ba24a7dfbddfL94

https://github.com/yiisoft/db-migration/blob/1e7a5173db3ad4c86aef6c123ab1e4398af35f9d/src/Service/MigrationService.php#L55-L59

Reply:
Moved `$this->migrator->setIO($io);` from `MigrationService` to `...Command::execute()`